pkg/scheduler/cache/cache.go

```diff
@@ -1225,8 +1225,16 @@ func (sc *SchedulerCache) GetMetricsData() {
 				if len(warnings) > 0 {
 					klog.V(3).Infof("Warning querying Prometheus: %v", warnings)
 				}
+				if res == nil || res.String() == "" {
+					klog.V(4).Infof("Warning querying Prometheus: no data found for %s", queryStr)
+					continue
+				}
 
+				klog.V(4).Infof("Query prometheus res %s", res.String())
 				rowValues := strings.Split(strings.TrimSpace(res.String()), "=>")
+				if len(rowValues) < 2 {
```

Comment:
I'm sorry for a little bit confused about this condition. Can you add some comments for that?

Reply:
done it.